test/dht_store_mutable.js

```diff
@@ -1,8 +1,8 @@
-const common = require('./common')
-const DHT = require('../')
-const ed = require('bittorrent-dht-sodium')
-const test = require('tape')
-const crypto = require('crypto')
+import ed from 'bittorrent-dht-sodium'
+import crypto from 'crypto'
```

Comment:
```suggestion
import crypto from 'node:crypto'
```

Also make it first?

Reply:
imports are sorted by module name for now shall we change that?

Reply:
Yes please!